pywwt/layers.py

```diff
@@ -374,6 +442,37 @@ def _check_cmap(self, proposal):
         else:
             return proposal['value']
 
+    @validate('time_att')
+    def _check_time_att(self, proposal):
+        # Parse the time_att column and make sure it's in the proper format
+        # (string in isot format, astropy Time, or datetime)
+        col = self.table[proposal['value']]
+
+        if (isinstance(col, STR_TYPE)
+            or np.issubdtype(col.dtype, NP_STR_TYPE)):
+
+            col_list = col.tolist()
+            is_iso = all(iso_test.match(t) for t in col_list)
```

Comment:
Note that you could also do the following if you want to avoid implementing the validation:

```python
try:
    Time(col, format='isot')
    return proposal['value']
except <exceptiontype>:
    raise ValueError('...')
```

And in future, we could then even start to support more string formats that astropy supports.

Reply:
This seems like a good call. It's switched in the next commit.